vm/chef/cookbooks/singlefs/files/get_metadata.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -eu
+#
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Get a metadata value from the metadata server.
+declare -r MDS_PREFIX=http://metadata.google.internal/computeMetadata/v1
+
+function print_metadata_value() {
```

Comment:
This script defines functions either already implemented or similar to what the c2d-config cookbook already offers: https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/vm/chef/cookbooks/c2d-config/files/c2d-utils. Please use the cookbook and if needed, extend it with additional features.